packages/create-plugin/templates/common/.config/Dockerfile

```diff
@@ -1,7 +1,12 @@
-ARG grafana_version=latest
```

Comment:
Think we still need a way to specify version. https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment#configure-the-grafana-version

Reply:
@sunker thank you for your review, we decided to use the non dev base images the same way as it was before, therefore we have again the ability to set grafana version. I've updated the PR to do that. This should also solve the issue with the e2e tests.